src/window.vala

```diff
@@ -38,7 +38,7 @@ public class WriteAs.MainWindow : Gtk.ApplicationWindow {
 
     construct {
         header = new Gtk.HeaderBar();
-        header.title = "Write.as";
+        header.title = _("Write.as");
```

Comment:
I don't know whether we may want to make this string translatable, because it seems to be a service name.

Reply:
Yes you're right